api/stripe.go

```diff
@@ -74,6 +72,21 @@ type (
 		ProductID   string  `json:"productId"`
 		LiveMode    bool    `json:"livemode"`
 	}
+	// SubscriptionInfoGET is a basic description of the subscription.
+	SubscriptionInfoGET struct {
```

Comment:
This is a subset of the Stripe's Subscription object. What gets in here is entirely up to @meeh0w, what I've put here for the moment is just a demo.